app/dao/api_key_dao.py

```diff
@@ -10,6 +10,14 @@
 from app.models import ApiKey
 
 
+@transactional
+def resign_api_keys():
```

Comment:
Let's add some Python documentation to say what's the intent of this method, we have the what with the code but we can add the why.